tests/contracts/sources.ts

```diff
@@ -285,4 +285,73 @@ export const contractSources: { [key: string]: string } = {
             chainid = block.chainid;
         }
     }`,
+  JoinCandidatesWrapper: `
```

Comment:
This is why we don't need the plain .sol file above. It is only used in this test, so this is the canonical location.